pycytominer/cyto_utils/cells.py

```diff
@@ -244,11 +256,33 @@ def load_image(self):
 
         """
 
-        # Extract image metadata
-        image_query = "select {} from image".format(
-            ", ".join(np.union1d(self.image_cols, self.strata))
-        )
+        image_query = "select * from image"
         self.image_df = pd.read_sql(sql=image_query, con=self.conn)
+
+        if self.add_image_features:
+            check_image_features(
+                self.image_feature_categories, list(self.image_df.columns)
+            )
+
+            image_features = list(
+                self.image_df.columns[
+                    self.image_df.columns.str.startswith(
+                        tuple(self.image_feature_categories)
+                    )
+                ]
+            )
+            image_features_df = self.image_df[image_features]
+            other_features_df = self.image_df[
+                list(np.union1d(self.image_cols, self.strata))
+            ]
+            image_features_df = image_features_df.add_prefix("Image_")
```

Comment:
In the past, I've added additional checks to this prefix to confirm that they aren't already prefixed. We would want to avoid a column name `Image_Image_CellCount` for example. 

```python
# Something like (I don't actually think this would work!)
image_features_df.columns = [f"Image_{x}" for x in image_features_df.columns if not x.startswith("Image_") else x]
```

Reply:
Good point - https://github.com/cytomining/pycytominer/pull/168/commits/f85f097a8aa9292d61e5a30c012f84d6e085cbc5